src/cmd-compress

```diff
@@ -29,8 +29,16 @@ parser.add_argument("--compressor",
                     choices=['xz', 'gzip'],
                     default=DEFAULT_COMPRESSOR,
                     help=f"Compressor to use, default is {DEFAULT_COMPRESSOR}")
+parser.add_argument("--fast", action='store_true',
+                    help="Compress with gzip -1")
```

Comment:
Nit: maybe `Override compression to gzip -1` to clarify the interaction with `--compressor`?

Reply:
Sure, changed.